### PR TITLE
new icon: filamentphp (original)

### PR DESCRIPTION
**Pull request by AnshSinghSonkhia**
# Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format new icon: Icon name (versions separated by comma). More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the develop branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the devicon.json file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes** #2120 

# Link to prove your SVG is correct and up-to-date.
https://filamentphp.com/

### CODE REVIEW

**Review by canaleal — CHANGES_REQUESTED** [review]
Nice, here are few changes that are needed:

- Instead of `plain-wordmark` call the SVG `original` as the icon you added is the original filamentphp logo. 
- Optimize and clean up the SVG like we did in the [remix pr](https://github.com/devicons/devicon/pull/2150#pullrequestreview-1905343380). Because the SVG only has 1 color, you can combine the paths into 1 path; This reduces the size of the svg and makes it possible for us the svg as an base/ alias for other version. 
- Because there is only 1 path and fill color you can use the `filamentphp-original.svg` as the base and make aliases for `original-wordmark`, `plain`, and `plain-wordmark`.

**Comment by AnshSinghSonkhia** [comment]
> * Instead of `plain-wordmark` call the SVG `original` as the icon you added is the original filamentphp logo.
> * Optimize and clean up the SVG like we did in the [remix pr](https://github.com/devicons/devicon/pull/2150#pullrequestreview-1905343380). Because the SVG only has 1 color, you can combine the paths into 1 path; This reduces the size of the svg and makes it possible for us the svg as an base/ alias for other version.

These changes are done!!

**Comment by AnshSinghSonkhia** [comment]
> * Because there is only 1 path and fill color you can use the `filamentphp-original.svg` as the base and make aliases for `original-wordmark`, `plain`, and `plain-wordmark`.

Made some changes in `devicon.json`

```json
"aliases": [
            {
                "base": "original",
                "alias": [
                    "original-wordmark",
                    "plain",
                    "plain-wordmark"
                ]
            }
        ]
```

Kindly. verify it @canaleal

File: devicon.json [hunk 1]
```diff
@@ -3145,6 +3145,32 @@
...
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#000000",
+        "aliases": [
```
Comment (canaleal) [inline devicon.json:3163]: Close but an array is not supported. Each alias will need to be its own object. 

```
        "aliases": [
            {
                "base": "original",
                "alias": "original-wordmark"
            }, 
             {
                "base": "original",
                "alias": "plain"
            },
             {
                "base": "original",
                "alias": "plain-wordmark"
            } 
        ]
```

Comment (AnshSinghSonkhia) [reply devicon.json:3163]: Okay, I thought it would save space & size :)

**Comment by canaleal** [comment]
By the way, the SVG has a `fill="currentColor"` and `class="transition duration-300 will-change-transform group-hover/filament:-translate-x-[1.2rem] motion-reduce:transition-none"` that can be removed.

**Comment by AnshSinghSonkhia** [comment]
> By the way, the SVG has a `fill="currentColor"` and `class="transition duration-300 will-change-transform group-hover/filament:-translate-x-[1.2rem] motion-reduce:transition-none"` that can be removed.

Done, anymore changes required?

**Review by canaleal — APPROVED** [review]
Perfect 👍 🦖 

I added any empty altnames array to the json and updated the svg paths a little so there are no line artifacts in icomoon

image

**Review by Snailedlt — APPROVED** [review]
Great use of aliases! 💯 
Thank you for your contribution! 🙇‍♂️

**Pull request merged**